dataprofiler/profilers/unstructured_text_profile.py

```diff
@@ -139,20 +140,12 @@ def _merge_words(self, other, merged_profile):
         :type merged_profile: TextProfiler
         :return:
         """
-        if not self._is_case_sensitive:
-            merged_profile.word_count = self.word_count.copy()
-            additive_words = other.word_count
-        else:
-            merged_profile.word_count = other.word_count.copy()
-            additive_words = self.word_count
-
-        for word in additive_words:
-            word_lower = word.lower()
-            if word_lower not in self._stop_words:
-                if self._is_case_sensitive:
-                    merged_profile.word_count[word] += additive_words[word]
-                else:
-                    merged_profile.word_count[word_lower] += additive_words[word]
+        merged_profile.word_count = self.word_count + other.word_count
```

Comment:
do we have a test validating cases of two diff sensitivities?

Reply:
yes, we already had

Reply:
Not both directions though right?

Reply:
I dont think that matters

Reply:
but I added another case for completeness

Reply:
It did matter because I believe the previous code actually would have failed said case.